src/components/ContentstackRichText/AnnotatedImageBlock/AnnotatedImageBlock.tsx

```diff
@@ -0,0 +1,58 @@
+import { css } from '@emotion/react';
+
+import { palette } from '@leafygreen-ui/palette';
```

Comment:
trying to avoid palette as much as possible - use token equivalent (or the closest like as instead)